src/appleseed-max-impl/appleseedsssmtl/appleseedsssmtl.h

```diff
@@ -178,8 +179,10 @@ class AppleseedSSSMtlBrowserEntryInfo
 
 class AppleseedSSSMtlClassDesc
   : public ClassDesc2
+  , IMtlRender_Compatibility_MtlBase
```

Comment:
Inherit publicly:

    public IMtlRender_Compatibility_MtlBase

This applies to other materials as well.